nats-tools/nas/dir_store.md

```diff
@@ -63,7 +63,7 @@ Success! - added account "B"
 With the account and a couple of users in place, let's push all the accounts to the nats-account-server:
 
 ```text
-> nsc push -A
+> nsc push -u http://localhost:9090/jwt/v1/ -A
```

Comment:
This is only required if the operator doesn't set an account server URL. 
To simplify the workflow, you can edit the operator:
```
❯ nsc edit operator -u http://localhost:9090/jwt/v1
[ OK ] set account jwt server url to "http://localhost:9090/jwt/v1"
[ OK ] edited operator "AAA"

```

Reply:
I concur with @aricart

Reply:
Thanks, understood. It was not clear when reading through the docs that this was necessary or preferred over the `push -u` option. That is why I wanted to raise it here. Of course feel free to update the docs however is appropriate.